scss/tech.scss

```diff
@@ -0,0 +1,54 @@
+.tech {
+  padding: 10px 40px;
+}
+
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  flex-basis: 150px;
+  left: 10px;
+  margin-bottom: 50px;
+  margin-right: 30px;
+  position: relative;
+  height: 280px;
+  width: 180px;
+
+  &::before {
+    bottom: 10px;
+    border: 1px solid $green;
+    border-radius: 5px;
+    content: '';
+    left: -10px;
+    right: 10px;
+    position: absolute;
+    top: -10px;
+    z-index: -1;
+  }
+}
+
+.tech-title,
+.tech-date,
+.tech-description,
+.tech-image {
+  margin: 10px 0;
```

Comment:
you just need bottom margins for this cases and the description should have no margin bottom since it's the last element